convex_hull.cpp

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Remove the sublime input comments from the files added.
And add more comments in the code to make the code more readable.